cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -89,6 +93,8 @@ class Home extends Component {
     };
 
     //By default, expect a single page -- update when search is performed and we can parse it
+    this.pageSize = 0;
```

Comment:
Shouldn't this be 1. We always want to show the contents of the first page right? The page dropdown right now, if there are no entities on the home page looks sort of broken.